yaml/morphology/extension.yaml

```diff
@@ -16,9 +16,9 @@ values:
     brief: spatiotemporal end
   - abbrev: GRA
     name: graduative
-    brief: increasing degree or quantity
+    brief: gradual/fading spatiotemporal beginning
```

Comment:
I don't think this follows the intent of the original.

> The GRADUATIVE extension is the inverse of the DEPLETIVE, focusing on a diffuse, extended “fade-in” or gradual onset of a noun. It would be used in translating the words darkness, wonder, and music in the following sentences: Darkness came upon us, I felt a growing sense of wonder, The music was very soft at first. With verbs it is illustrated by verbs and phrases such as to fade in, to start gradually, to build up, and similar notions, e.g., She’s been eating more and more lately.  

‘the headwaters of a river’
‘a gradually developing situation’

I don't think there's an analogy between DEL:PRX::ICP:GRA. Though I agree it's kind of "set up" to seem that way, I think it's not the _focus of discussion_ that fades in, but the thing itself being spoken about (e.g. "darkness", "wonder", "music", "she's been eating more and more").

Reply:
> I don't think there's an analogy between DEL:PRX::ICP:GRA.

Oh certainly not. I view DEL and PRX as contrasting between each other, and then ICP, ATV, GRA, and DEL as the combinations of two binary variables (start vs. end and instant vs. gradual).

> I think it's not the focus of discussion that fades in, but the thing itself being spoken about (e.g. "darkness", "wonder", "music", "she's been eating more and more").

I changed it from "increasing degree or quantity" because GRA also conveys that the thing is beginning (gradually).